### PR TITLE
Set galaxy_uwsgi_static_conf to false by default.

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

**Comment by mvdbeek** [comment]
@bgruening don't merge yet, I realized there are two more defaults missing.

**Comment by mvdbeek** [comment]
@bgruening OK, this should be good now. Sorry for the mess.

File: defaults/main.yml [hunk 1]
```diff
@@ -53,6 +53,11 @@ galaxy_database_connection: "postgres://{{ galaxy_user_name }}@localhost:{{ gala
 galaxy_uwsgi: true
 uwsgi_port: 4001
 
+# Set to true to write thread/process number into supervisor config. If false, uses environmental variables.
+galaxy_uwsgi_static_conf: false
+# number of uwsgi threads to use if galaxy_uwsgi_static_conf is true
+uwsgi_threads=2
```
Comment (bgruening) [inline defaults/main.yml]: Does this work?
`uwsgi_threads: 2`

**Comment by bgruening** [comment]
Galaxy Docker gained travis testing recently. I find it useful to test these changes against this container to make sure most of the stuff is still working. Maybe you want to try that.

File: defaults/main.yml (hunk 1 above)
Comment (mvdbeek) [inline defaults/main.yml]: you're right, this wouldn't have worked.

**Comment by mvdbeek** [comment]
Oh yes, this looks simple enough!
So I fork docker-galaxy-stable, and register my repo with travis?

**Comment by bgruening** [comment]
Yes. And than update the submodule to track your fork or simply add it by hand :)
Sorry to urge you into this ...

**Comment by mvdbeek** [comment]
No worries, I should be doing this for another project too.

**Comment by mvdbeek** [comment]
@bgruening it's still running, but I see the bioblend tests passing
https://travis-ci.org/mvdbeek/docker-galaxy-stable
This is awesome!

**Comment by bgruening** [comment]
Wonderful! Yeah this is going to rock. I included FTP testing as well a few days ago, thinking about other tests as well. Would be nice to test Jupyter (IE) in Galaxy or even the outside SLURM scheduler.

**Comment by mvdbeek** [comment]
Me and @fabiorjvieira are working on setting up instances completely and exclusively with ansible. This is almost ready to go public, and it also works in docker. That means we could also test the various roles we have been using (galaxy-extras, galaxy-tools, galaxy-os, galaxy, ...) ... hmmm, this is fun!

**Comment by bgruening** [comment]
:+1:

**Comment by bgruening** [comment]
So I see your tests have passed! If you don't oppose I will merge this.

**Comment by mvdbeek** [comment]
Sure, go ahead. Thanks!

**Comment by bgruening** [comment]
Thanks Marius!

**Pull request merged**